content/blog/2018/12/2018-12-10-outreachy-audit-log-plugin.adoc

```diff
@@ -0,0 +1,23 @@
+---
+:layout: post
+:title: Outreachy internships to add audit logging support to Jenkins
+:tags:
+- community
+- outreachy
+- outreachy2018
+:author: jvz
+---
+This year marks the first time the Jenkins Foundation is participating in https://www.outreachy.org/[Outreachy].
+Outreachy is a program vaguely similar to Google Summer of Code (GSoC) where interns work on open source projects for a paid stipend.
+The key difference is that Outreachy reaches out to underrepresented minorities and those who face systemic bias or discrimination in the technology industry in their home country.
+Once I learned about this program, I immediately volunteered to mentor as the concept strongly aligns with my ideals of inclusiveness and community building.
+I'm happy to report that both the Jenkins Foundation and CloudBees have been very supportive of this program.
+
+Expanding on our previous efforts to mentor students in GSoC, this year we've joined up with Outreachy to mentor two interns this year.
+Joining with me to implement https://github.com/jenkinsci/audit-log-plugin[audit logging support for Jenkins] are https://github.com/Lathaguna[Latha Gunasekar] and https://github.com/davidolorundare[David Olorundare].
+These two will be learning a lot about both professional software engineering as well as open source community engagement.
```

Comment:
```suggestion
I am excited to welcome both David and Latha, and am looking forward to what they will learn about both professional software engineering and contributing to an open source community.
```

Hope you don't mind me putting some words in your mouth :zipper_mouth_face:

Reply:
LGTM!